docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -27,6 +27,9 @@ license: |
 
 `ALTER TABLE RENAME TO` statement changes the table name of an existing table in the database.
 
+If the table is cached, the commands clear cached data of the table and all its dependents that refer to it. The cache will be lazily filled when the next time the table or the dependents are accessed.
```

Comment:
`The cache` -> `The cache of the table being renamed (not its dependents)`

Reply:
The next sentence says that: "The partition rename command keeps table dependents as cached but the table rename does not."

Reply:
It's better to make it clear at the beginning, instead of correcting it later.

Reply:
The problem is the doc section mixes 2 different commands. I would split the description completely like:

If the table is cached:
- the table rename command ...
- the partition rename command ...

Reply:
yea that's even better!